controllers/usersController.js

```diff
@@ -48,6 +48,25 @@ const getUser = async (req, res) => {
   }
 }
 
+/**
+ * Checks whether a user exists or not
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const checkUser = async (req, res) => {
```

Comment:
The function name should also be changed accordingly

Reply:
done